Documentation/commonServerConstants.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "language": "python",
+    "origin": "CommonServerPython",
+    "cmdType": "builtinVar",
+    "return_value": {},
+    "description": "A dictionary containing all possible entry types - note, file, error, image, etc (e.g. entryTypes['note'])",
```

Comment:
shouldn't we list the entire options for it (instead of writing "etc")?

Reply:
I listed the ones that we actually use.  I'm not sure we want to overly expose things like downloadAgent,pinned, userManagement, playgroundError, entryInfoFile, and map that I'm not even sure what they mean.
I'll better descriptions though